config.go

```diff
@@ -64,6 +66,16 @@ type NutsGlobalConfig struct {
 	v *viper.Viper
 }
 
+// NutsConfigValues exposes global configuration values
```

Comment:
Is this interface used?

Reply:
I'm using it in the registry to mock the config